lib/factories/ppt/index.js

```diff
@@ -50,13 +50,34 @@ class PptFactory {
         return this.slideFactory.addSlide(slideName, layoutName);
     }
 
-    addImage(slide, filePath, options) {
+    addImage(slide, filePath, options = {}) {
         let mediaName = `image-${uuidv4()}${path.extname(filePath)}`;
         let rId = this.slideFactory.addImageToSlideRelationship(slide, `../media/${mediaName}`);
 
+        if (options.url) {
+            // if this is not a link to another slide
+            if (options.url[0] !== '#') {
+                // interestingly enough, you can't just give PowerPoint a simple URL like "www.google.com" - it
+                // MUST contain the protocol prefix; so we'll put "http://" if the caller didn't specify it
+                if (!options.url.startsWith('http')) {
+                    options.url = `http://${options.url}`;
```

Comment:
i think we should default to `https` i think it's more secure and the way the world is trending. What do you think?

Reply:
Fine by me.